pkg/printers/.import-restrictions

```diff
@@ -0,0 +1,10 @@
+{
+  "Rules": [
+    {
+      "SelectorRegexp": "k8s[.]io/kubernetes/pkg/(api$|apis/)",
```

Comment:
Sanity check, you're matching `pkg/api` and `pkg/apis/<foo>`, but not the root  `pkg/apis`, correct?